lesson/day01.md

```diff
@@ -5,103 +5,109 @@ title: Day 1
+The first tool you will need to create assembly programs is a source code editor. Some 
+popular editors are listed below along with their supported operating systems.
 
 1. Notepad
     - OS Support: Windows
-    - Included with every installation of Windows, but not many features.
+    - Included with every installation of Windows, but does not have many features.
 2. Notepad++
     - OS Support: Windows
     - Lots of features, and easy to use. 
     - Recommended for inexperienced programmers.
 3. Geany
     - OS Support: Windows, Linux
-    - Similar featureset to Notepad++ while being cross platform.
-    - only depends on GTK+ without further toolkit dependancys.
+    - Has similar features to Notepad++
+    - Only depends on GTK+ without further toolkit dependancies.
 4. Vim
     - OS Support: Windows, Mac, Linux
     - The hacker's editor. Extremely powerful, but has a very steep learning curve.
 
-ED NOTE: Editor recommendations welcome.
-
-### Emulator
-
-Once you've written a program, you need a way to run it. Simply transferring
-it to your calculator and running it there is one option (which we'll discuss
-later), but it becomes very hard to debug programs running on a calculator
-(and there will be bugs in your programs). For that reason, we'll use an
-emulator, which is a program that runs on your computer and acts like a
-calculator would.
-
-Our emulator of choice here is [WabbitEmu](https://wabbit.codeplex.com/), as
-it is the best emulator that is still actively developed. All you have to do
-is install and run it. To use the emulator, you'll need a ROM image,a file
-containing a complete copy of your calculator's software, the process of
-obtaining which is known as 'ROM dumping'. Please note that obtaining a ROM
-image in a way other than dumping your calculator's ROM to the computer is
-illegal. The tool of choice for a ROM dump is [rom8x](http://www.ticalc.org/archives/files/fileinfo/373/37341.html).
-TiLP can also perform a ROM dump, but rom8x is far more diverse in usability.
-
-To dump your ROM, find the folder named after your calculator. For example, a
-TI-84+CSE would be under 84C, a TI-83+ would be under 83+, and a TI-83+ Silver
-Edition would be under 83S. Send those two .8xp files to your calculator, and
-run them one at a time. You might need to clear out some RAM space, though.
-Send the AppVar created by the program under its name to your computer. Then,
-download the latest OS upgrade file for your model on [TI's website](http://education.ti.com/).
-Then, in the Command Prompt window, navigate to the folder with rom8x.exe,
-the two AppVars, and the OS upgrade file, and type this in:
+**[ED NOTE]: Editor recommendations welcome.**
+
+## Assembler
+
+An assembler translates source code into machine code that the calculator can execute. In
+this guide we will use an assembler called [Brass](http://www.benryves.com/bin/brass/). 
+**[CAPTAIN CALC EDIT]: Should the Doors SDK also be mentioned here?**
```

Comment:
Maybe we need to build our own SDK package.. Then setup is trivial.

Reply:
Good idea! That would help beginners get up and running quickly.